eng/pipelines/templates/jobs/perf.yml

```diff
@@ -37,6 +40,7 @@ resources:
     type: github
     endpoint: Azure
     name: Azure/azure-sdk-tools
+    ref: refs/pull/3547/merge
```

Comment:
Temporary?

Reply:
Yes, this is temporary to validate the changes.  Will be removed when 3547 is merged to main.